organizing_hub/templates/calls/partials/call_campaign_callers_explainer.html

```diff
@@ -0,0 +1,4 @@
+<p>
```

Comment:
I didn't wanna put this in a config because it seemed weird but not sure if this makes the most sense either.

Reply:
Generally I think this is ok. Ideally all of our content strings would be in some sort of messages list, and not hardcoded into templates. This way we can separate content from template logic. Also helps for translation/multi-language support. Then we would pull content from the messages list by a key.

However I do see that you are including `p` tags inside the partial, but in one area you also have `p` tags around the call to partial, and another area you don't. So need to be consistent here.

Reply:
👍